stable/etcd-operator/templates/crds.yaml

```diff
@@ -0,0 +1,6 @@
+{{- if .Values.customResources.createDefinition }}
```

Comment:
i think we should keep this to `.Values.deployments.etcdOperator` for determining if this should be created. Since we are also going to just create a single file, we can be explicit here about the file to get.